libbeat/docs/newdashboards.asciidoc

```diff
@@ -18,74 +21,210 @@ export the Kibana dashboards.
-  -h | -help
-    Print the help menu.
-  -d | -dir
-    Local directory where the dashboards, visualizations, searches and index 
-    pattern are saved.
-  -l | -url
-    Elasticseacrh URL. By default is http://localhost:9200.
-  -u | -user
-    Username and password for authenticating to Elasticsearch using Basic
-    Authentication. The username and password should be separated by a
-    colon (i.e. admin:secret). By default no username and password are
-    used.
-  -i | -index
-    Kibana index pattern where to save the dashboards, visualizations,
-    index patterns. By default is .kibana.
-
+  -beat string
+       	Specify the Beat name, in case the archive contains the dashboards for multiple Beats.
+  -dir string
+       	Directory containing the subdirectories: dashboard, visualization, search, index-pattern. eg. kibana/
+  -es string
+       	Elasticsearch URL (default "http://127.0.0.1:9200")
+  -file string
+       	Zip archive file containing the Beats dashboards.
+  -i string
+       	Overwrites the Elasticsearch index name. For example you can replaces metricbeat-* with custombeat-*
+  -k string
+       	Kibana index (default ".kibana")
+  -only-dashboards
+       	Import only dashboards together with visualizations and searches. By default imports both, dashboards and the index-pattern.
+  -only-index
+       	Import only the index-pattern. By default imports both, dashboards and the index pattern.
+  -pass string
+       	Password to connect to Elasticsearch
+  -url string
+       	URL to the zip archive containing the Beats dashboards (default "https://download.elastic.co/beats/dashboards/beats-dashboards-5.0.0-alpha6.zip")
+  -user string
+       	Username to connect to Elasticsearch
 ----------------------------------------------------------------------
 
-==== dir
-The input directory that contains the dashboards and all the dependencies. The default value is the current directory.
 
-==== url
+==== es
 The Elasticsearch URL. The default value is http://localhost:9200.
 
 ==== user
-The username and password for authenticating the connection to Elasticsearch by using Basic Authentication. The username
-and password should be separated by a colon. By default no username and password are used.
+The username for authenticating the connection to Elasticsearch by using Basic Authentication. By default no username and password are used.
 
-==== kibana
+
+==== pass
+The password for authenticating the connection to Elasticsearch by using Basic Authentication. By default no username and password are used.
+
+==== k
 The Elasticsearch index pattern where Kibana saves its configuration. The default value is `.kibana`.
 
+==== i
+The Elasticsearch index pattern where the Beat pushes the data. You must use this option only if you want to change the
+index pattern name than the default one.
+
+==== only-dashboatds
+If true, then only the dashboards are imported together with dependencies: visualizations, searches. By default is
```

Comment:
"If true" makes it sound like you need to set the option to true or false. 

I'd say something like: "If specified, then only the dashboards, along with their visualizations and searches, are imported.  If not specified, the default is to import everything, including the dashboards and the index pattern."